docs/architecture/adr-063-privval-grpc.md

```diff
@@ -0,0 +1,75 @@
+# ADR 063: Privval gRPC
+
+## Changelog
+
+- 23/11/2020: Initial Version (@marbar3778)
+
+## Context
+
+Validators use remote signers to help secure their keys. This system is Tendermint's recommended way to secure validators, but the path to integration with Tendermint's private validator client is plagued with custom protocols. 
+
+Tendermint uses its own custom secure connection protocol (`SecretConnection`) and a raw tcp/unix socket connection protocol. The secure connection protocol until recently was exposed to man in the middle attacks and limits the amount and speed of integrations users can make. The raw tcp connection protocol is less custom, but has been causing minuet issues with users. 
+
+Migrating Tendermint's private validator to a widely adopted protocol will ease the current maintenance and integration burden experienced with the current protocol. 
```

Comment:
By widely adopted we mean that most kms's use gRPC? and so no custom socket is needed to use the kms right?

Reply:
Will reword. Meant the migration to gRPC will make it easier to for integrators and maintainers of kms's.